src/root.component.tsx

```diff
@@ -19,9 +39,97 @@ export default function Root(props) {
   return (
     <>
       <GlobalStyles />
-      <Notice>{props.name} is mounted!</Notice>
       <MadieEditor {...outputProps} />
       <div data-testid="madie-editor-value">{editorVal}</div>
+      <Nav>
+        <InnerNav>
+          <InnerMost>
+            <Logo href="/">
+              <Image>
```

Comment:
I think ideally the SVG would be a separate file, imported via webpack

Reply:
Done